app.js

```diff
@@ -31,6 +34,15 @@ app.get('/getalledges', async function (req, res) {
   })
 })
 
+app.post('/:getallteas', async function (req, res) {
+  console.log(req.body)
+  // pass json into funtion 
+  TeaService.addTea(req.body)
+  // .then((result) => {
```

Comment:
Uncomment

Reply:
and add different status codes. 200 for added, another for missing data,  and default 500